### DISCUSSION TITLE
how to remove /projects ?

### DISCUSSION THREAD

**Discussion by arastoonet** [trevortylerlee/astro-micro#16] [Q&A] [answered]
I don't need and have /projects instead I need an about page. how can I remove /projects?

**Comment by trevortylerlee** [comment, marked as answer]
To add an "About" page, create a new file in `src/pages` called `about.astro`. Use `src/pages/index.astro` as a starting point and delete what isn't needed.

To remove the `/projects` collection:
1. Delete the `PROJECTS` metadata from `src/consts.ts`
2. Delete the `projects` collection in `src/content/config.ts`
3. Delete the `projects` directory in `src/content`
4. Delete the `projects` directory in `src/pages/projects`
5. Delete the `Link` to `/projects` in `src/components/Header.astro`, specifically lines 20 to 23
6. Delete the Projects section in `/src/pages/index.astro`, specifically lines 84 to 100
